chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature." + "\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp:
```

Comment:
oh that makes a lot of sense. I believe there was one other command that could use something like that too. I'll look through the commands and make changes to aliases as appropriate.